### ISSUE TITLE
error with data augmentation

### ISSUE THREAD

**Issue by huatangzhi** [chsasank/Traffic-Sign-Classification.keras#5]
Epoch 1/30
31367/31367 [==============================] - 979s 31ms/step - loss: 0.2193 - acc: 0.9354 - val_loss: 0.0298 - val_acc: 0.9916
Epoch 2/30
31367/31367 [==============================] - 969s 31ms/step - loss: 5.1823 - acc: 0.5994 - val_loss: 15.9742 - val_acc: 0.0089
Epoch 3/30
 8834/31367 [=======>......................] - ETA: 11:48 - loss: 15.9145 - acc: 0.0126

after epoch1 the loss seems bigger and bigger

**Comment by boyang-yu** [comment]
Yes we have exactly the same problem like yours. So could you please tell me how did you do to solve this problem? Thank you!
